content/en/blog/_posts/2024-04-23-recursive-read-only-mounts.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blog
+title: 'Kubernetes 1.30: Read-only volume mounts can be finally literally read-only'
+date: 2024-04-23
+slug: recursive-read-only-mounts
+---
+
+**Author:** Akihiro Suda (NTT)
+
+Kubernetes 1.30 added alpha support for "recursive read-only mounts".
+
+Prior to this release, read-only volume mounts were surprisingly not really read-only under specific conditions.
```

Comment:
This is a scary statement. I think I would say something like:

"""
Read-only volume mounts have been a feature of Kubernetes since 1.0.  Surprisingly, read-only mounts are not completely read-only under certain conditions.  As of the v1.30 release, they can be made completely read-only.
"""

Reply:
Technically, not "since 1.0"
https://github.com/kubernetes/kubernetes/blob/v0.2/pkg/api/v1beta1/types.go#L124

Maybe just s/since 1.0/since the beginning/ ?

Reply:
Reworded as
```
Read-only volume mounts have been a feature of Kubernetes since the beginning.
Surprisingly, read-only mounts are not completely read-only under certain conditions.
As of the v1.30 release, they can be made completely read-only,
with alpha support for _recursive read-only mounts_.
```